hdr_html_canvas_element.md

```diff
@@ -125,13 +125,23 @@ The component signals are mapped to red, green and blue tristimulus values accor
 * Transfer function:
 
 ```
-   E = | E' / 12.92, if abs(E') ≤ 0.04045
-       | ((E' + 0.055) / 1.055)^2.4, otherwise
+decoding equation (non-linear to linear):
+
+if N < -0.04045
+  L = -1.0 * (((-1.0*N) + 0.055) / 1.055)^2.4;
+elseif (N >= -0.04045 && N <= 0.04045)
+  L  = N / 12.92;
+else
+  L = ((N + 0.055) / 1.055)^2.4;
+end
 
-       with E' ∈ ℝ
 ```
 
-where `E'` is the non-linear colour value and `E` is the linear colour value
+where `N` is the non-linear colour value and `L` is the linear colour value
+
+![extended-srgb-decoding-function](https://user-images.githubusercontent.com/37905569/121105535-25ae7700-c7b9-11eb-92fd-a90e68cac7f3.png)
```

Comment:
IIUC embedded images should be uploaded into this repo (similar to what's done in [PR 61](https://github.com/w3c/ColorWeb-CG/pull/61)).

If we are to include an illustration of transfer functions, I'd prefer to have a single figure with all transfer functions included on it, something to the effect of [this](https://colab.research.google.com/drive/1UJvajR9mh7u5lWWzHMIyncQFsiIZykRm?usp=sharing). I'm not convinced about how helpful the illustration would be.